Podfile

```diff
@@ -35,7 +35,7 @@ end
 
 def wordpress_kit
   # To allow pod to pick up beta versions use -beta. E.g., 1.1.7-beta.1
-  pod 'WordPressKit', '~> 4.49.0'
+  pod 'WordPressKit', '~> 5.0.0'
```

Comment:
Unrelated to this PR, more of a question about what the standard for our internal libraries should be.

What do you think of using 

`~> 5.0-beta` here, so that `bundle exec pod update` or `bundle exec pod update WordPressKit` will automatically pick up any 5.x version, including betas? (Possibly `~> 5.0` with no beta, or some mechanism to remove `-beta` at code freeze time).

Or, should we keep using the current, stricter, approach and let devs and release managers update the minor, patch, and beta versions as necessary?

Reply:
I'd say using a stricter approach by default, and update this version requirement when we want to bring in newer version? I don't have strong opinion on this though. 😸